dracut/modules.d/80kiwi-verity/README.rst

```diff
@@ -0,0 +1,75 @@
+kiwi-parse-verity
+=================
+
+Overview
+--------
+
+This tool is used to parse the information stored by
+`Kiwi <https://osinside.github.io/kiwi/>`__ inside the unused space of a
+disk partition when using a SquashFS root filesystem backed by
+``dm-verity``. This data is stored at a known offset (generally 4096
+bytes, or 1 block, from the end of the partition) and follows the layout
+as described in the `Kiwi
+documentation <https://osinside.github.io/kiwi/image_description/elements.html#preferences-type>`__
+for the attribute ``embed_verity_metadata``.
+
+Building
+--------
+
+Rust and Cargo are required to build the tool. Simply call
+``cargo build`` and the tool and its dependencies will be built.
+
+Alternatively, a .deb can be built using the ``debuild`` command.
```

Comment:
`debbuild`

Reply:
This README was taken from a standalone Debian package that I had created, so `debuild` was correct here. It's however not really relevant anymore, so I'll just remove that line.